lib/protocol/bep.pb.go

```diff
@@ -297,6 +297,7 @@ type FileInfo struct {
 	Permissions   uint32       `protobuf:"varint,4,opt,name=permissions,proto3" json:"permissions,omitempty"`
 	ModifiedS     int64        `protobuf:"varint,5,opt,name=modified_s,json=modifiedS,proto3" json:"modified_s,omitempty"`
 	ModifiedNs    int32        `protobuf:"varint,11,opt,name=modified_ns,json=modifiedNs,proto3" json:"modified_ns,omitempty"`
+	LastModBy     string       `protobuf:"bytes,12,opt,name=lastmodby,proto3" json:"lastmodby,omitempty"`
```

Comment:
I don't see where the change is to bep.proto, but this should almost certainly be a `protocol.DeviceID` field, which is now possible natively since my last change. Don't forget the same change in lib/db/structs.proto